### PR TITLE
fix ltx includes in examples

### PR DESCRIPTION
**Pull request by chris-rock**
fixes ltx includes in examples

### CODE REVIEW

File: examples/server-and-client.js [hunk 1]
```diff
@@ -4,11 +4,11 @@ var xmpp = require('../index')
   , c2s = null
   , debug = require('debug')('server-and-client')
   , Client = require('node-xmpp-client')
-  , ltx = require('ltx')
+  , ltx = require('node-xmpp-core').ltx
 
 var startServer = function(done) {
     // Sets up the server.
-    c2s = new xmpp.C2SServer({
+    c2s = new xmpp.WebSocketServer({
```
Comment (sonnyp) [inline examples/server-and-client.js]: You sure about that?

Comment (chris-rock) [inline examples/server-and-client.js]: That looks wrong ;-) I push a fix.

**Comment by chris-rock** [comment]
rebased and fixed. thanks for the feedback @sonnyp

**Comment by chris-rock** [comment]
thanks @sonnyp

**Pull request merged**